features/currencies/currency_switching.feature

```diff
@@ -57,4 +57,4 @@ Feature: Currency selection
         And I change the currency to "GBP"
         And I added product "PHP Top" to cart, with quantity "1"
         When I go to the cart summary page
-        Then I should see item with unit price "£6.83" in the list
+        Then I should see item with price "£6.83" in the list
```

Comment:
I made Behat contexts a little bit more strict. Earlier, "price" or "unit price" was just a name, that didn't change nothing. Moreover, test was passing if given value only existed on the page. Now it uses that name to find suitable column, and then checks if there's given value in given column.